src/global.css

```diff
@@ -1,9 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Overlock&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Cookie&display=swap');
```

Comment:
Maybe you could combine these two: 
@import url('https://fonts.googleapis.com/css2?family=Cookie&family=Overlock&display=swap');